app/errors.py

```diff
@@ -0,0 +1,19 @@
+class VaccineError(Exception):
+    def __str__(self) -> str:
+        return "All friends should be vaccinated"
```

Comment:
It would be better to return this specific string in the required place directly, because `VaccineError` can apply to not only friends but to other people or animals as well out of the current context. Besides that, all inherited exceptions would have this text, and the vaccine absence can be not the only reason for this error. Replace this code with `pass`.